src/sonic_ax_impl/__main__.py

```diff
@@ -75,6 +76,10 @@ def install_fragments():
     sonic_ax_impl.logger.setLevel(log_level)
     ax_interface.logger.setLevel(log_level)
     swsssdk.logger.setLevel(log_level_sdk)
+    # Note: swsscommon uses a fixed log level for simplicity
+    # TODO: improve it when log_level option is used
+    log_level_swsscommon = swsscommon.Logger.SWSS_ERROR
```

Comment:
can this be modified as below:
if args.get('log_level') is not None and args.get('log_level') ==  logging.INFO:
    log_level_swsscommon  = swsscommon.Logger.SWSS_INFO
if args.get('log_level') is not None and args.get('log_level') ==  logging.ERROR:
    log_level_swsscommon  = swsscommon.Logger.SWSS_ERROR
else:
    log_level_swsscommon  = swsscommon.Logger.SWSS_ERROR

Reply:
I think `if args.get('log_level') is not None` is not needed and we can directly check the value of `args.get('log_level')`


Qi, what are we achieving with this change (in order to fix Azure/sonic-buildimage#5992). Will the warnings start appearing as errors?

Reply:
Can the description of the PR be modified with some context?

Reply:
Warnings in swsscommon will still be warnings. Here I set the log_level higher to ERROR, so warning will disappear in syslog.

---
In reply to: [529841456](https://github.com/Azure/sonic-snmpagent/pull/178#discussion_r529841456) [](ancestors = 529841456)

Reply:
Updated.

---
In reply to: [529842115](https://github.com/Azure/sonic-snmpagent/pull/178#discussion_r529842115) [](ancestors = 529842115)